src/common/util.js

```diff
@@ -608,6 +608,10 @@ function getShebang(text) {
   return text.slice(0, index);
 }
 
+/**
+ * @param {any} object
+ * @returns {object is Array<any>}
```

Comment:
I'm not familiar with JSDoc, but should we use `unknown`?

Reply:
Let's keep `any` for now